cli/root.go

```diff
@@ -41,7 +42,7 @@ var (
 	httpConfigFile  string
 	versionCheck    bool
 
-	configFiles = []string{os.ExpandEnv("$HOME/.config/amtool/config.yml"), "/etc/amtool/config.yml"}
+	configFiles = configdir.SystemConfig("amtool", "config.yml")
```

Comment:
This could introduce a change for existing users. I'd rather use `os.UserHomeDir()` to be platform agnostic and keep the same directory structure.

```suggestion
        var configFiles []string
	homeDir, err := os.UserHomeDir()
	if err == nil {
		configFiles = append(configFiles, filepath.Join(homeDir, ".config", "amtool" "config.yml")
	}
	configFiles = append("/etc/amtool/config.yml")
```